environment.yml

```diff
@@ -35,6 +35,8 @@ dependencies:
   - nbconvert>=5.4.1
   - nbsphinx
   - pandoc
+  - dask
```

Comment:
could this be an optional dependency?

Reply:
This is for the dev env. We've been including all the dependencies necessary to build the docs.

Reply:
must have been some misunderstanding in https://github.com/pandas-dev/pandas/pull/27646#issuecomment-518292531. That was a dev-only dependency.

Reply:
> This is for the dev env. We've been including all the dependencies necessary to build the docs.

That said, would it be possible to rely on just dask-core (+ what is needed for dask.dataframe), as distributed brings in a lot more dependencies? 
(the one code block that shows the client could be a code-block)